test-network/scripts/createChannel.sh

```diff
@@ -3,15 +3,19 @@
 # imports  
 . scripts/envVar.sh
 . scripts/utils.sh
+export PATH="${PWD}"/../../fabric/build/bin:"${ROOTDIR}"/../bin:"${PWD}"/../bin:"$PATH"
```

Comment:
are you sure this is needed?

Reply:
You need this in case someone will want to use old binaries.
Me and Dave had the same approach when constructing nano-bash

Reply:
Most people that download the samples and try it out will not have a fabric directory. I'd suggest to remove this change.
We added it for the nano bash sample since a target user of that sample was a Fabric developer who compiles their own binaries.